src/current/cockroachcloud/plan-your-cluster-basic.md

```diff
@@ -54,7 +54,7 @@ When you create a multi-region {{ site.data.products.basic }} cluster, you will
 You cannot currently remove regions once they have been added.
 {{site.data.alerts.end}}
 
-For optimal performance, deploy client applications in one of your cluster's configured regions. CockroachDB {{ site.data.products.basic }} uses a geolocation routing policy to automatically route clients to the nearest region, even if that region is not one of your cluster's configured regions. This means that if you are running an application from a region that is not used by your cluster, connecting to that region may cause high network latency. This may be acceptable for development, but should be avoided for any production or performance-sensitive applications. Refer to the [CockroachDB {{ site.data.products.basic }} FAQs]({% link cockroachcloud/serverless-faqs.md %}#how-do-i-get-the-sql-endpoint-for-a-specific-region-of-my-multi-region-cluster) for information on overriding the automatic routing policy.
```

Comment:
This link text seems to suggest that the FAQ has information that isn't in this document — should we find that elsewhere in the docs and link it?

Reply:
Yep, planning to move the content to the plan/manage cluster docs when I get to them -- made a note of it :)